packages/core/src/main/Dropdown/DropdownItemsGroup/storybook/template.tsx

```diff
@@ -1,12 +1,12 @@
 import { FC } from 'react';
 import {
-    DropdownItem,
+    DropdownItem as QuarXDropdownItem,
```

Comment:
чисто визуально приятнее, когда не смешиваются прописные и строчные буквы в имени)

```suggestion
    DropdownItem as QxDropdownItem,
```